labs/smartbft.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Hyperledger Fabric SmartBFT Library
+parent: Labs
+
+---
+# Lab Name
+
+[SmartBFT] - https://github.com/SmartBFT-Go/consensus
+
+
+# Short Description
+
+Hyperledger Fabric is an enterprise-grade, permissioned distributed ledger platform that offers modularity for a broad set of industry use cases. One modular component is a pluggable ordering service that establishes consensus on the order of transactions and batches them into blocks. This proposal suggests integrating the SmartBFT consensus library into Hyperledger Labs to enhance its blockchain frameworks with a robust, scalable, and resilient consensus mechanism designed for practical BFT deployment.
+
+This lab provides a BFT consensus library based on the [BFTSmart](https://github.com/bft-smart/library) protocol and written in Go. The library is tailored to the blockchain use case yet general enough to cater to a wide variety of other uses. The BFT library's design and integration into Fabric address crucial aspects that were left unsolved in all prior work, making it unfit for production use. 
+
+
+# Scope of Lab
+
+SmartBFT is a consensus library developed as part of the SmartBFT-Go project. It offers advancements in Byzantine Fault Tolerance with an emphasis on modularity, performance, and security. Its integration can address various challenges in distributed ledger technology, especially fault tolerance and scalability.
+The scope of this lab is to provide a BFT consensus library for the Hyperledger Fabric project. The library presents a design that addresses all the BFT system's concerns.
+
+The library suggests an easy integration with the Hyperledger Fabric, though still agnostic to the Fabric's internals implementation details, suggestion interfaces that capture the special needs of a blockchain application but are fully customizable for other use cases as well.
+
+Finally, the library was accepted into Hyperledger Fabric with [RFC-006](https://github.com/hyperledger/fabric-rfcs/blob/main/text/006-bft-based-ordering-service.md) and is now delivered within the next version of the Hyperledger Fabric 3.0. The purpose of suggesting moving the SmartBFT library into Labs is to consolidate work and governance under Hylerledger and Linux Foundation projects. 
+
+# Initial Committers
+
+* Yacov Manevich - [yacovm@il.ibm.com](mailto:yacovm@il.ibm.com)
+* Hagar Meir - [hagar.meir@ibm.com](mailto:hagar.meir@ibm.com)
+* Artem Barger - [bartem@il.ibm.com](mailto:bartem@il.ibm.com)
+* Yoav Tock - [tock@il.ibm.com](mailto:tock@il.ibm.com)
```

Comment:
Can you make sure to provide GitHub IDs for these folks as they will need to be invited to the lab maintainer team on GitHub?

Reply:
myself, and @HagarMeir , @C0rWin , and @tock-ibm

Reply:
fix